micro-rdk-server/.cargo/config.toml

```diff
@@ -1,20 +0,0 @@
-[target.xtensa-esp32-espidf]
```

Comment:
I removed this file because as far as I know, cargo doesn't consult `.cargo/config.toml` files from subdirectories of a workspace.